File: backend/schema/finanzas.py
from pydantic import BaseModel, Field, EmailStr
from typing import Optional, List


class Comprobante(BaseModel):
    id_evento: int
    nombre:str
    correo:EmailStr
    concepto:str
    monto:float
    fecha:str


class Pago(BaseModel):
    monto:float
    descripcion:str
    categoria:int


class Gasto(Pago):
    pass


class Transaccion_class(BaseModel):
    monto: float
    id_categoria: int
    descripcion: str
    categoria: int

File: Parroquia/src/app/models/finanzas.ts
export interface RegFinanza{
    monto: number,
    descripcion: string,
    id_categoria:number,
    evidencia: string | null
}

export interface GetAllFinanzas extends RegFinanza{
    id_transaccion: number,
    fecha: string,
    categoria: string,
    tipo_categoria: string,
}

